runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1911,6 +1974,32 @@ class INTERPRETER_CLASS
 						rc = THROW_CRIU_SINGLE_THREAD_MODE;
 						break;
 #endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
+#if JAVA_SPEC_VERSION >= 24
+					case J9_OBJECT_MONITOR_YIELD_VIRTUAL:
+					{
+						rc = EXECUTE_BYTECODE;
```

Comment:
is the code here identical to line 1822 - 1847? we should turn that into a function

Reply:
we discussed this in our code review on Fri. in the call, this was the helper function @fengxue-IS plan's to add by EOD.

Reply:
okay, sounds good